### ISSUE TITLE
Plugin not recognized

### ISSUE THREAD

**Issue by chrisgrieser** [TfTHacker/obsidian42-brat#4]
I was trying to install [limelight](https://github.com/smikula/obsidian-limelight) via BRAT, but BRAT says that it is no plugin due to a missing `manifest.json`.
<img width=60% alt="Screenshot 2021-10-16 14 28 41" src="https://user-images.githubusercontent.com/73286100/137587446-9cec4344-cfc3-4c85-ab45-79e97f9373fe.png">

However, the plugin does have a `manifest.json`, as can be seen [here](https://github.com/smikula/obsidian-limelight/releases).

**Comment by TfTHacker** [comment]
Yes, this plugin is not configured properly. It will  also fail to load if added to obsidian community plugin list.

All plugins require a manifest.json in the root folder of the repository, in addition to the manifest.json being in the Release for the plugin.

This plugin is missing the manifest.json in the root, hence the error from BRAT.

So in this case you as a beta tester have already found a bug to report to the Limelight dev :-)
